lib/send-json.js

```diff
@@ -7,8 +7,17 @@
 
 const safeStringify = require('fast-safe-stringify');
 
-module.exports = function sendJson(res, data) {
-  const content = safeStringify({error: data});
+module.exports = function sendJson(res, data, options) {
+  options = options || {};
+  let err = data;
+  if (options.rootProperty == null) {
+    err = {error: data};
+  } else if (typeof options.rootProperty === 'string') {
+    if (options.rootProperty) {
+      err = {[options.rootProperty]: data};
+    }
+  }
```

Comment:
These conditions are very difficult to understand, it's not clear that `rootProperty: false` is intentionally supported.

Proposed solution:

```js
const err = options.rootProperty === false
  ? data
  : {[options.rootProperty || 'error']: data};
```

I think it would be best to extract this bit into a function that can be shared with the XML serializer too.